dts/bindings/adc/atmel,sam-afec.yaml

```diff
@@ -31,5 +31,53 @@ properties:
         For example the I2C on SAME7x would be
            pinctrl-0 = <&pa8b_afec0_adtrg>;
 
+    dmas:
+      required: false
+      description: |
+        DMA specifiers.  Each specifier will have a phandle
+        reference to the dma controller, the channel number, and peripheral
+        trigger source.
+
+        For example dmas for ADC0 channel 0:
+          dmas = <&xdmac 0 DMA_PERID_AFEC0_RX>;
+
+    trigger-source:
+      type: int
+      required: false
+      default: 0
+      enum:
+        - 0 # Software driven
+        - 1 # TIOA output of Timer Counter channel 0 for AFEC0
+            # TIOA output of Timer Counter channel 3 for AFEC1
+        - 2 # TIOA output of Timer Counter channel 1 for AFEC0
+            # TIOA output of Timer Counter channel 4 for AFEC1
+        - 3 # TIOA output of Timer Counter channel 2 for AFEC0
+            # TIOA output of Timer Counter channel 5 for AFEC1
+        - 4 # PWM0 event line 0 for AFEC0
+            # PWM1 event line 0 for AFEC1
+        - 5 # PWM0 event line 1 for AFEC0
+            # PWM1 event line 1 for AFEC1
+        - 6 # Analog Comparator
```

Comment:
If you put this in the description, it will show up in the bindings index:

https://docs.zephyrproject.org/latest/reference/devicetree/bindings.html

I would suggest that because that way users do not have to navigate to the binding file on the file system to look up this useful information you have gathered. But it is not a blocking request.